src/resources/views/crud/columns/upload.blade.php

```diff
@@ -0,0 +1,11 @@
+@php
```

Comment:
Hmmm the link didn't really work for me - error 404.

It points to:
```
http://bp-2022-10-11-columns.test/storage/uploads/monsters/upload_field/e71266adfe0b4e940adf163793ba0e51.png
```

But it should be
```
http://bp-2022-10-11-columns.test/uploads/monsters/upload_field/e71266adfe0b4e940adf163793ba0e51.png
```

(no storage)

Please take a look at the `upload_multiple` column and how we get the URL there. We should get it the same way here, so it works.

Additionally, I no longer think we should use the `download` attribute... since I see `upload_multiple` doesn't use that. It's nice but... we have to keep consistency between `upload` and `upload_multiple`.

Reply:
Removed `download` attribute & `uploads` from href